scripts/generateDocs.js

```diff
@@ -1,39 +1,48 @@
-const path = require('path');
-const markdownMagic = require('markdown-magic');
-const fs = require('fs');
-const https = require('https');
+const path = require("path");
+const markdownMagic = require("markdown-magic");
+const fs = require("fs");
+const https = require("https");
```

Comment:
Curious, why were all the quotes changed?

Reply:
I have "format on save" turned on and it was doing weird things. Reverted changes and tried again.